projects/addon-charts/components/axes/axes.component.ts

```diff
@@ -42,7 +42,7 @@ export class TuiAxesComponent {
 
     @Input()
     @tuiDefaultProp()
-    axisYLabels: readonly string[] = [];
+    axisYLabels: readonly (string | null)[] = [];
```

Comment:
I believe only `axisXLabels` has special case for `null` to hide ticks, all other arrays don't care for `null`.

Reply:
fixed